contrib/docker/Dockerfile.debian

```diff
@@ -0,0 +1,55 @@
+# -*-Dockerfile-*-
+
+ARG DEBIAN_VERSION=buster
+ARG LDC_VERSION=1.17.0
+
+FROM debian:${DEBIAN_VERSION} AS builder-onedrive
+ARG LDC_VERSION
+
+RUN apt-get update \
+ && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends build-essential curl ca-certificates libcurl4-openssl-dev libsqlite3-dev libxml2-dev pkg-config \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN ARCH="$(dpkg --print-architecture)" \
+ && case "${ARCH}" in \
+      aarch64|arm64) \
+        BINARY_URL="https://github.com/ldc-developers/ldc/releases/download/v${LDC_VERSION}/ldc2-${LDC_VERSION}-linux-aarch64.tar.xz"; \
+        ;; \
+      armhf|arm) \
+        BINARY_URL="https://github.com/ldc-developers/ldc/releases/download/v${LDC_VERSION}/ldc2-${LDC_VERSION}-linux-armhf.tar.xz"; \
+        ;; \
+      amd64|i386:x86-64) \
+        BINARY_URL="https://github.com/ldc-developers/ldc/releases/download/v${LDC_VERSION}/ldc2-${LDC_VERSION}-linux-x86_64.tar.xz"; \
```

Comment:
@MarcelCoding 
The x86_64 version of LDC wont run on i386 | i686 .. and, in the past, there were many issues with i686 platforms due to compiler linkage. 

Can you drop i386 ?

Reply:
Sure, I actually tested x86 and noticed that it isn't working. I removed it from github actions and missed the Dockerfile.